crates/fluvio-extension-common/src/output/mod.rs

```diff
@@ -57,13 +57,13 @@ mod error {
 }
 
 pub trait DisplayTable {
-    fn print_std(&self);
+    fn print_std(&self, indent: u8);
 }
 
 impl DisplayTable for Table {
-    fn print_std(&self) {
```

Comment:
This break semversion guarantee.  So for now, we should keep `print_std` and mark as deprecate

Reply:
Not sure what you mean, I think that `print_std` was created in this PR